x-pack/plugins/endpoint/public/embeddables/resolver/view/process_event_dot.tsx

```diff
@@ -21,40 +27,78 @@ import * as selectors from '../store/selectors';
 const nodeAssets = {
   runningProcessCube: {
     cubeSymbol: `#${SymbolIds.runningProcessCube}`,
-    labelBackground: NamedColors.fullLabelBackground,
+    labelBackground: NamedColors.labelBackgroundRunningProcess,
     descriptionFill: NamedColors.empty,
     descriptionText: i18n.translate('xpack.endpoint.resolver.runningProcess', {
       defaultMessage: 'Running Process',
     }),
   },
   runningTriggerCube: {
     cubeSymbol: `#${SymbolIds.runningTriggerCube}`,
-    labelBackground: NamedColors.fullLabelBackground,
+    labelBackground: NamedColors.labelBackgroundRunningTrigger,
     descriptionFill: NamedColors.empty,
     descriptionText: i18n.translate('xpack.endpoint.resolver.runningTrigger', {
       defaultMessage: 'Running Trigger',
     }),
   },
   terminatedProcessCube: {
     cubeSymbol: `#${SymbolIds.terminatedProcessCube}`,
-    labelBackground: NamedColors.fullLabelBackground,
+    labelBackground: NamedColors.labelBackgroundTerminatedProcess,
     descriptionFill: NamedColors.empty,
     descriptionText: i18n.translate('xpack.endpoint.resolver.terminatedProcess', {
       defaultMessage: 'Terminated Process',
     }),
   },
   terminatedTriggerCube: {
     cubeSymbol: `#${SymbolIds.terminatedTriggerCube}`,
-    labelBackground: NamedColors.fullLabelBackground,
+    labelBackground: NamedColors.labelBackgroundTerminatedTrigger,
     descriptionFill: NamedColors.empty,
     descriptionText: i18n.translate('xpack.endpoint.resolver.terminatedTrigger', {
       defaultMessage: 'Terminated Trigger',
     }),
   },
 };
 
+const ChildEventsButton = () => {
+  return (
+    <EuiButton
+      onClick={(clickEvent: React.MouseEvent<HTMLButtonElement, MouseEvent>) => {
+        clickEvent.preventDefault();
+        clickEvent.stopPropagation();
+      }}
+      color="ghost"
+      size="s"
+      iconType="arrowDown"
+      iconSide="right"
+      tabIndex={-1}
+    >
+      {i18n.translate('xpack.endpoint.resolver.relatedEvents', {
+        defaultMessage: 'Events',
+      })}
+    </EuiButton>
+  );
+};
+
+const RelatedAlertsButton = () => {
+  return (
+    <EuiButton
+      onClick={(clickEvent: React.MouseEvent<HTMLButtonElement, MouseEvent>) => {
```

Comment:
can you wrap the functions in useCallback, this way `EuiButton` will get the same function reference each time `RelatedAlertsButton` is rendered, and `EuiButton` won't necessarily have to rerender just because its parent did.

```suggestion
<EuiButton
  onClick={useCallback((clickEvent: React.MouseEvent<HTMLButtonElement, MouseEvent>) => {
    clickEvent.preventDefault();
    clickEvent.stopPropagation();
  }, [])}
```